gantry/firmware/spi.c

```diff
@@ -111,13 +111,21 @@ SPI_HandleTypeDef MX_SPI2_Init() {
     return hspi2;
 }
 
+void get_SPI_handle(void* tmp){
```

Comment:
Hold on. I don't think we need this file anymore. At this point it duplicates what happens in `common/firmware/spi/spi.c`, and we don't need the duplication to capture the right handle anymore because it's now passed in to `hal_transmit_receive`. I think this file can go away.

Reply:
And then add the same code to gantry `can_task` as was added in the pipette and head subprojects